### PR TITLE
Update all v3 EAs to latest version

### PR DESCRIPTION
**Pull request by alejoberardino**

### CODE REVIEW

File: .changeset/dry-dancers-pretend.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+'@chainlink/nft-blue-chip-adapter': minor
+'@chainlink/nomics-test-adapter': minor
+'@chainlink/oanda-adapter': minor
+'@chainlink/polkadot-balance-adapter': minor
+'@chainlink/tiingo-test-adapter': minor
+'@chainlink/tradermade-test-adapter': minor
+'@chainlink/twosigma-adapter': minor
+---
+
+Update v3 framework to latest version
```
Comment (amit-momin) [inline .changeset/dry-dancers-pretend.md:29]: Think the changeset is missing `blocksize-capital-test` and `moonbeam-address-list`

Comment (alejoberardino) [reply .changeset/dry-dancers-pretend.md:29]: done

**Comment by amit-momin** [comment]
Don't have a place to pin this comment but I noticed references to `customSettings` in the generate-readme script that may need to be updated

File: packages/sources/nft-blue-chip/src/transport/marketcap.ts [hunk 2]
```diff
@@ -106,93 +102,25 @@ export interface MarketcapTransportConfig {
 
 // Much of the code in this transport is borrowed from the framework's RestTransport as an example
 export class MarketcapTransport implements Transport<MarketcapTransportGenerics> {
-  inFlightPrefix!: string
-  cache!: Cache<AdapterResponse<MarketcapTransportGenerics['Response']>>
-  inFlightCache!: Cache<boolean>
-  rateLimiter!: RequestRateLimiter
+  name = 'marketcap'
```
[the remark this answers is not in the document]
Comment (alejoberardino) [reply packages/sources/nft-blue-chip/src/transport/marketcap.ts]: I'll change this a bit after this is merged so it's not necessary to do it (or it is, but it's more strict) but for now I've changed it to the default

File: packages/sources/oanda/src/transport/modifiedSSE.ts [hunk 3]
```diff
@@ -29,6 +29,7 @@ export type ModifiedSSEConfig = {
 }
 
 export class ModifiedSseTransport<T extends TransportGenerics> extends StreamingTransport<T> {
+  name = 'default_single_transport'
```
Comment (amit-momin) [inline packages/sources/oanda/src/transport/modifiedSSE.ts:32]: I think this one should be `sse` like it was before since this adapter has a router

**Review by amit-momin — APPROVED** [review]

**Comment by alejoberardino** [comment]
I will test tomorrow, wish these boys luck

**Pull request merged**